client/my-sites/themes/themes-selection.jsx

```diff
@@ -143,3 +154,34 @@ export default connect(
 		};
 	}
 )( ThemesSelection );
+
+class ThemesSelectonWithPage extends React.Component {
+
+	constructor() {
+		super();
+		this.state = {
+			page: 1,
+		};
```

Comment:
Nit -- for simple initial state like this, I've adopted to writing

```js
class ThemesSelectonWithPage extends React.Component {
  state = {
    page:1
  }

/* methods */
}
```

as recommended by https://babeljs.io/blog/2015/06/07/react-on-es6-plus. (I think hat doc is becoming de-facto standard for Calypso)